custom_components/tuya_local/devices/loratap_wifi_curtain_switch_double.yaml

```diff
@@ -0,0 +1,175 @@
+name: Loratap Wifi Double Curtain Switch
+products:
+  - id: zqypumy8ot4clzyi
+primary_entity:
+  entity: cover
+  name: Curtain
+  class: blind
+  dps:
+    - id: 1
+      name: control
+      type: string
+      mapping:
+        - dps_val: open
+          value: open
+        - dps_val: close
+          value: close
+        - dps_val: stop
+          value: stop
+    - id: 2
+      name: position
+      type: integer
+      unit: "%"
+      range:
+        max: 100
+        min: 0
+      mapping:
+        - step: 1
+secondary_entities:
+  - entity: cover
+    name: Curtain 2
+    dps:
+      - id: 4
+        name: control
+        type: string
+        mapping:
+          - dps_val: open
+            value: open
+          - dps_val: close
+            value: close
+          - dps_val: stop
+            value: stop
+      - id: 5
+        name: position
+        type: integer
+        unit: "%"
+        range:
+          max: 100
+          min: 0
+        mapping:
+          - step: 1
+  - entity: number
+    name: Position 1
+    category: config
+    icon: "mdi:blinds-horizontal-closed"
+    dps:
+      - id: 2
+        name: value
+        type: integer
+        unit: "%"
+        range:
+          max: 100
+          min: 0
+        mapping:
+          - scale: 0
+          - step: 1
+  - entity: number
+    name: Position 2
+    category: config
+    icon: "mdi:blinds-horizontal-closed"
+    dps:
+      - id: 5
+        name: value
+        type: integer
+        unit: "%"
+        range:
+          max: 100
+          min: 0
+        mapping:
+          - scale: 0
+          - step: 1
+  - entity: number
+    name: timer
+    category: config
+    icon: "mdi:timer"
+    dps:
+      - id: 10
+        name: value
+        type: integer
+        range:
+          max: 120
+          min: 2
+        mapping:
+          - scale: 0
+          - step: 1
+  - entity: number
+    name: timer 2
+    category: config
+    icon: "mdi:timer"
+    dps:
+      - id: 11
+        name: value
+        type: integer
+        range:
+          max: 120
+          min: 2
+        mapping:
+          - scale: 0
+          - step: 1
+  - entity: select
+    name: direction
+    category: config
+    icon: "mdi:arrow-u-down-left"
+    dps:
+      - id: 8
+        name: option
+        type: string
+        mapping:
+          - dps_val: "forward"
+            value: forward
+          - dps_val: "back"
+            value: back
+  - entity: select
+    name: direction 2
+    category: config
+    icon: "mdi:arrow-u-down-left"
+    dps:
+      - id: 9
+        name: option
+        type: string
+        mapping:
+          - dps_val: "forward"
+            value: forward
+          - dps_val: "back"
+            value: back
+  - entity: select
+    name: Light Mode
+    category: config
+    icon: "mdi:toggle-switch"
+    dps:
+      - id: 14
+        name: option
+        type: string
+        mapping:
+          - dps_val: "relay"
+            value: Mode 1
```

Comment:
Mode 1, Mode 2 is not very descriptive.  Suggest you look at similar devices for the mappings used there.  "relay" normally means the light follows the switch position, "pos" means always on as a locator for the switch when it is dark, and "none" means always off.  But I'm not entirely sure how "relay" maps to curtain switches.

Reply:
I just copied the text from the app, they call it Mode1 and Mode2, but even when i change them in the App, the do change in tuya-local but i see no behaviour change on the switch itself. Not entire sure what they do. I wondered if it just disables the relay in the switch to stop it clicking (ie if you're not using the physical electrical connections)

Reply:
They may have taken a standard wall-switch template from Tuya, and left off the LEDs for button backlights. Normally in a typical Tuya wall switch with touch-sensitive buttons, there will be an LED backlit ring around each button, and this controls whether this is always lit, never lit, or lights up when the switch is on.  If it does nothing, just add it as an attribute on the corresponding cover entity, as it isn't useful to have as a separate entity.